### ISSUE TITLE
"\\" expected to break line

### ISSUE THREAD

**Issue by carloshgsilva** [amake/flutter_tex_js#3]
Code:
```dart
TexImage(r'1+5+3 \\ 2*3', displayMode: false)
```
Console:
```
Escaped text to render; was: "1+5+3 \\ 2*3"; escaped: "1+5+3 \\\ 213"
```
Expected to work the same as:
```dart
TexImage(r'1+5+3 \newline 2*3', displayMode: false)
```
Potential solution:
replace
https://github.com/amake/flutter_tex_js/blob/1b710e44af0c01422976caac144275caa289997c/lib/flutter_tex_js.dart#L72
with
```dart
 final _unescapedBackslashPattern = RegExp(r'\\'); 
```

I didn't really understand the use of the current regex, feel free to comment if I missed anything.

**Comment by amake** [comment]
The intent of the regex was to escape `\`, but only when it's not already escaped. Basically it's [this scenario](https://stackoverflow.com/a/11819111/448068).

The reason we need to escape it is because the LaTeX markup gets stuffed into a JavaScript string literal to be evaluated in the native webview. Example:

- LaTeX string: `\alpha`
- Interpreted as JavaScript string literal: `alpha` (invalid)
- Escaped: `\\alpha`
- Escaped version interpreted as JavaScript string literal: `\alpha` (as intended)

But now that I think about it, I guess there isn't a reason to be so picky about what we escape. We're doing a round-trip: escape for JavaScript, then unescape by evaluating. So everything needs to be escaped.

**Comment by amake** [comment]
This should be fixed in v0.1.6, now released.
